packages/core/src/module.js

```diff
@@ -149,7 +149,7 @@ export default class ModuleContainer {
     }
 
     // Prevent adding buildModules-listed entries in production
-    if (this.options.buildModules.includes(handler) && this.options._start) {
+    if (handler && this.options.buildModules.includes(handler) && this.options._start) {
```

Comment:
We are silently ignoring an important issue here. If handler is not resolved at this point, at least a warning should be shown (it is not same situation of using a build-module but a module not loaded from config because of typo)